### ISSUE TITLE
x in non container

### ISSUE THREAD

**Issue by aroberge** [friendly-traceback/friendly-traceback#142]
See https://twitter.com/andre_roberge/status/1549900130769330176

**Comment by alexmojaki** [comment]
`executing` can already identify the problematic `x in y` node, so it should be easy to tell what is happening and produce a good targeted message.

**Comment by aroberge** [comment]
Yes, I should have noted that in this issue - as we can see in this screenshot:
![image](https://user-images.githubusercontent.com/629698/183194160-f7ed43ab-a2b2-4159-874b-00292d3be96f.png)

`executing` is **magical**.

**Comment by aroberge** [comment]
Other than the translation, this has been taken care of. Closing the issue.

**Comment by aroberge** [comment]
Reopening the issue base on feedback from Alex in a previous commit. The wrong object is identified in the following case:
```
[1]: class B: pass
...:

[2]: b, b2 = B(), B()

[3]: b2 is (b if False else b2)
True

[4]: 'a' in (b if False else b2)

Traceback (most recent call last):
  Code block [4], line 1
    'a' in (b if False else b2)
TypeError: argument of type 'B' is not iterable

[5]: why()

    An iterable is an object capable of returning its members one at a time.
    Python containers (`list, tuple, dict`, etc.) are iterables.
    'b' is not a container. A container is required here.
```
`b2` should have been identified as the problematic object.  As Alex mentioned, I need to use the AST to extract the value of the right hand side argument, so that this works correctly.
